PaloAltoCortexXDR/CHANGELOG.md

```diff
@@ -7,8 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## 2024-01-23 - 1.14.0
+## 2024-01-23 - 1.0.0
 
 ### Added
 
-- Add for the first time Cortex Query alerts
\ No newline at end of file
+- Add for the first time Cortex Query alerts
+
+## 2024-02-01 - 1.0.1
+
+### Change
+
+- Improve the timestamp setter ( Add one seconde to the last timestamp )
```

Comment:
Could you reverse the order? The most recent entry should be the first one.

Reply:
yes sorry for that